proxy/api/src/process.rs

```diff
@@ -112,7 +112,7 @@ async fn run_rigging(
 
     if let Some(seeds_sender) = seeds_sender {
         let seeds_store = ctx.store().clone();
-        coco::SpawnAbortable::new(async move {
+        let _seeds_event_task = coco::SpawnAbortable::new(async move {
```

Comment:
This should be awaited in the select down below, then we don't run into the same issue again in the future.